src/api.js

```diff
@@ -341,7 +341,7 @@
             callbacks.success(data);
           }
         },
-        failure: callbacks.failure
```

Comment:
Otherwise this throws an error: `agent.setConfiguration(config, { success: () => { /* ... */ } });`